WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1311,7 +1311,7 @@ ComposeViewSubjectInputField= border:none; color:@TxtC@; height:3rem; @FullWidth
 ComposeViewTinyMceContainer =
 ComposeViewHtmlEditor       = padding:@ComposeViewOutsidePadding@; background-color: @AppC@;
 ComposeViewPlainTextEditor  = @FontSize-big@ @FontFamily-fixed@ @FullWidth@ @AppBorder@ overflow:auto; padding:8px;
-ComposeViewHeaderEmptyRight = padding-right: 18px;
+ComposeViewHeaderEmptyRight = padding-right: 18px; width: 1%;
```

Comment:
not sure but empty space would be reserved for scroll bar area, can we change it to px or em?